random_question/weighted_random.py

```diff
@@ -10,11 +10,26 @@
 
 # TODO: Hess should implement!
```

Comment:
take care to delete obsolete comments. in general, you should avoid righting comments, **in favour of writing very concise and readable code**. 
Write comments only as a last resort and delete obsolete ones.

Reply:
done